go/private/context.bzl

```diff
@@ -75,6 +75,19 @@ load(
     "get_source",
 )
 
+CPP_TOOLCHAIN_TYPE = Label("@bazel_tools//tools/cpp:toolchain_type")
+CGO_ATTRS = {
```

Comment:
should we expose these to `defs.bzl`? (Is the intention for extending rules that use `go_context` to use these instead of the previous mechanism?)